query-engine/driver-adapters/package.json

```diff
@@ -17,9 +17,9 @@
   "author": "",
   "devDependencies": {
     "@types/node": "20.10.4",
-    "tsup": "8.0.1",
-    "typescript": "5.3.3",
     "esbuild": "0.19.8",
-    "esbuild-register": "3.5.0"
+    "tsup": "8.0.1",
+    "tsx": "^4.7.0",
```

Comment:
Lesson learnt: it seems that every `devDependency` declared in `prisma/prisma`'s pnpm workspace root needs to be re-declared in `query-engine/driver-adapters` (see how actions [fail](https://github.com/prisma/prisma-engines/actions/runs/7818510935/job/21329490096?pr=4697) when that's not the case).

Replacing `esbuild-register` with `tsx` was necessary after https://github.com/prisma/prisma/pull/23003.